setup.py

```diff
@@ -18,5 +18,5 @@
     packages=find_packages(exclude=[]),
     version="0.2.1",
```

Comment:
```suggestion
    version="0.2.2",
```
Let's bump the version and do a release to be sure it's working correctly.